include/mata/nfa-plumbing.hh

```diff
@@ -39,10 +39,10 @@ namespace Plumbing {
             Nfa*               result,
             const Nfa&         aut,
             const Alphabet&    alphabet,
-            const StringMap&  params = {{"algorithm", "classical"}},
-            std::unordered_map<StateSet, State> *subset_map = nullptr)
+            const StringMap&  params = {{"algorithm", "classical"},
+                                        {"minimize", "false"}})
```

Comment:
rename the parameter minimize to brzozowski? Because it is not just adding minimization, it is running different subset constructions which may explode in different ways ...

Reply:
I think we should just remove the `params` argument as was suggested by @tfiedor. Maybe we should just have enum that decide what type of algorithm to run (so either classical, or classical_with_minimization, or some possibly other one). But it should probably be another PR.

Reply:
For now, this conversation is considered to be resolved. We will continue with this discussion later on in a new issue.